development-docs/DEV_GUIDE.md

```diff
@@ -51,8 +51,8 @@ To build Strimzi from source the operator and Kafka code needs to be compiled in
 4. To use the newly built images, update the `install/cluster-operator/060-Deployment-strimzi-cluster-operator.yml` to obtain the images from your repositories on Docker Hub rather than the official Strimzi images:
 
     ```
-    sed -Ei -e "s#(image|value): strimzi/([a-z0-9-]+):latest#\1: $DOCKER_ORG/\2:latest#" \
-            -e "s#([0-9.]+)=strimzi/([a-zA-Z0-9-]+:[a-zA-Z0-9.-]+-kafka-[0-9.]+)#\1=$DOCKER_ORG/\2#" \
+    sed -Ei -e "s#(image|value): quay.io/strimzi/([a-z0-9-]+):latest#\1: docker.io/$DOCKER_ORG/\2:latest#" \
+            -e "s#([0-9.]+)=quay.io/strimzi/([a-zA-Z0-9-]+:[a-zA-Z0-9.-]+-kafka-[0-9.]+)#\1=docker.io/$DOCKER_ORG/\2#" \
```

Comment:
Should this use `$DOCKER_REGISTRY` instead of `docker.io`?

Reply:
I have updated the PR with the above changes.